lib/ansible/modules/cloud/amazon/cloudtrail.py

```diff
@@ -262,33 +262,29 @@
             sample: {'environment': 'dev', 'Name': 'default'}
 '''
 
-import traceback
-
 try:
-    from botocore.exceptions import ClientError
+    from botocore.exceptions import ClientError, BotoCoreError
 except ImportError:
-    # Handled in main() by imported HAS_BOTO3
-    pass
+    pass  # Handled by AnsibleAWSModule
 
-from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.ec2 import (boto3_conn, ec2_argument_spec, get_aws_connection_info,
-                                      HAS_BOTO3, ansible_dict_to_boto3_tag_list,
-                                      boto3_tag_list_to_ansible_dict, camel_dict_to_snake_dict)
+from ansible.module_utils.aws.core import AnsibleAWSModule
+from ansible.module_utils.ec2 import (get_aws_connection_info, camel_dict_to_snake_dict,
```

Comment:
`get_aws_connection_info` shouldn't be needed in this module, region should be available from `module.params`.

Reply:
Only when region is *explicitly* passed in.  Otherwise it can be pulled from the environment or even defaults to us-east-1.

Doing anything other than pulling it via get_aws_connection_info would likely introduce a breaking change.  This is also why I wanted to add #66988

Reply:
migrated to module.region